content/media-player/visualize/visualize_1.md

```diff
@@ -0,0 +1,39 @@
++++
+title = "Visualize your data"
+weight = 1
+post = ""
++++
+
+Now that you have modeled data you would probably want to see the results in a visual way as well to be able to easily to identify patterns and trends in your data.
+
+We have created a Tableau workbook to showcase how you could do that, using csvs that were exported from the derived core web vitals tables (`snowplow_media_player_media_stats`, `snowplow_media_player_plays_by_pageview`, `snowplow_media_player_media_ads`, `snowplow_media_player_media_ad_views`).
+
+If you would like to see your own modeled data you can simply change the data sources and then you will be able to look at the visualizations to get inspiration of what you can build with your connector of choice (based on the warehouse you have).
+
+#### Accessing the sample Tableau workbook
+
+You can download the file [here](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/media_player/snowplow_media_player_tableau_demo.twbx).
+
+In case you need the source csvs, you can access them here:
+* [snowplow_media_player_media_stats](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/media_player/snowplow_media_player_media_stats.csv)
+* [snowplow_media_player_plays_by_pageview](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/media_player/snowplow_media_player_plays_by_pageview.csv)
+* [snowplow_media_player_media_ads](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/media_player/snowplow_media_player_media_ads.csv)
+* [snowplow_media_player_media_ad_views](https://snowplow-demo-datasets.s3.eu-central-1.amazonaws.com/media_player/snowplow_media_player_media_ad_views.csv)
+
+Once you have the file locally, you can open it in several ways: with Tableau Public, Tableau Reader (free) or by uploading it to Tableau Online, if you have access.
+
+#### Change the dashboard data source
+
+1. Export your `snowplow_media_player_media_stats`, `snowplow_media_player_plays_by_pageview`, `snowplow_media_player_media_ads` and `snowplow_media_player_media_ad_views` derived tables as csvs with your sql editor of choice.
+
+2. In Tableau Public or Tableau Online open the sample twbx file.
+3. Right-click the data source you wish to edit in the data pane.
+4. From the menu, select Edit Connection.
+5. In the Edit Connection dialog box, you have two options:
+    - Navigate to the new location of the data source by browsing your computer's file system.
+    - Select a new data source altogether by clicking on the Browse button and choosing a different file.
+6. After you have selected the new location or data source, select OK to save the changes.
+
+You should have a working dashboard with your own data! Feel free to pick and choose visualizations and dashboards to make your custom one.
```

Comment:
I would add a few introductory sentences to explain what each of the dashboards /charts do. I'm tired atm and did not even notice the multiple tabs at first! It would also help them understand what we are trying to show easily (e.g Retention chart)

Reply:
Also, is that too late to make fine-tunings on the dashboard itself? The content is really nice, i just feel like the tables blend in the background, it could do with a pop of colours perhaps by adding some stat bar charts behind the numbers within at least one column in the tables or colouring the header row purple perhaps: 
<img width="1066" alt="image" src="https://github.com/snowplow-incubator/data-product-accelerators/assets/95634439/6d39b0ee-a4e0-4204-82a1-04e7ee31dee8">

Reply:
Good points I'll get these added in  🙌  and definitely not too late for dashboard changes, any suggestions welcome!